packages/notion-types/src/collection.ts

```diff
@@ -46,7 +46,7 @@ export interface Collection {
     }>
     property_visibility?: Array<{
       property: PropertyID
-      visibility: 'show' | 'hide'
+      visibility: 'show' | 'hide' | 'hide_if_empty'
```

Comment:
Hey @nabettu,
Adding the def type doesn't actually hide the properties on the page. Is this supposed to make that change?

Reply:
oh Sorry I add only type.
I will change collection-row.tsx for this property. plz wait.